src/main/java/IncomeTaxCalculator.java

```diff
@@ -3,12 +3,14 @@
 
 public class IncomeTaxCalculator {
 
+    //configurable?
```

Comment:
Can you think of a way of making the tax-year configurable rather than hard-coding it?
Also, is this the best name?